eclipse-scout-core/src/style/colors.less

```diff
@@ -286,10 +286,11 @@
 @menu-item-disabled-color: @disabled-color;
 @messagebox-background-color: @popup-background-color;
 @messagebox-body-color: @palette-black;
-@mode-hover-text-color: @hover-color;
-@mode-selected-background-color: @selected-background-color;
-@mode-selected-text-color: @selected-color;
-@mode-alternative-selected-color: @mode-selected-background-color;
+@mode-text-color: @palette-gray-7;
+@mode-hover-text-color: @mode-text-color;
+@mode-selected-text-color: @default-button-color;
+@mode-alternative-selected-background-color: @button-background-color;
+@mode-alternative-background-color: #ebebeb;
```

Comment:
Please use a color from the palette or add a todo marker so that I can adjust it once the color palette is final